eth_worker/eth_manager/task_interfaces/blockchain_sync/blockchain_sync_constants.py

```diff
@@ -0,0 +1,228 @@
+THIRD_PARTY_SYNC_FILTERS = 'THIRD_PARTY_SYNC_FILTERS'
+THIRD_PARTY_SYNC_JOBS = 'THIRD_PARTY_SYNC_JOBS'
+MAX_ENQUEUED_BLOCK = '%s_MAX_ENQUEUED_BLOCK'
+BLOCKS_PER_REQUEST = 1000
+ERC20_ABI =  """
+[
```

Comment:
ERC20 abis are already stored in.... ummm *checks repo..
here!
https://github.com/teamsempo/SempoBlockchain/blob/master/eth_worker/eth_manager/ABIs/erc20_abi.py